compiler/testData/diagnostics/tests/annotations/rendering/memberProjectedOut.fir.kt

```diff
@@ -12,5 +12,5 @@ class C<T> {
 }
 
 fun test(a: C<out CharSequence>) {
-    <!INAPPLICABLE_CANDIDATE!>a[1] = 25<!>
+    <!INAPPLICABLE_CANDIDATE!>a[1]<!> = 25
```

Comment:
How did `INAPPLICABLE_CANDIDATE` change in this PR?

Reply:
Maybe because `source` of convention set/get is changed in this PR?

Reply:
Ah I missed that change, and your comment about `SET` using parent source before. Thanks for clarifying.